inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/KafkaLoadNode.java

```diff
@@ -157,6 +173,12 @@ public Map<String, String> tableOptions() {
                 if (StringUtils.isNotBlank(topicPattern)) {
                     options.put("topic-pattern", topicPattern);
                 }
+                if (StringUtils.isNotBlank(sinkPartitioner)) {
+                    options.put("sink.partitioner", sinkPartitioner);
+                }
+                if (StringUtils.isNotBlank(partitionPattern)) {
+                    options.put("sink.multiple.partition-pattern", partitionPattern);
```

Comment:
Maybe, optional option  can be passed by properties object. we don't need change construction method.

Reply:
Mainly to facilitate api integration calls, and for compatibility, a new constructor has been added.